packages/nx/src/command-line/activate-key/command-object.ts

```diff
@@ -2,16 +2,14 @@ import { CommandModule } from 'yargs';
 import { withVerbose } from '../yargs-utils/shared-options';
 import { handleErrors } from '../../utils/handle-errors';
 
-export interface ActivatePowerpackOptions {
-  license: string;
+export interface ActivateKeyOptions {
+  key: string;
   verbose: boolean;
 }
 
-export const yargsActivatePowerpackCommand: CommandModule<
-  {},
-  ActivatePowerpackOptions
-> = {
-  command: 'activate-powerpack <license>',
+export const yargsActivateKeyCommand: CommandModule<{}, ActivateKeyOptions> = {
```

Comment:
Can we have 1 command? `nx register` 

Aliases are `nx activate-powerpack`.. and `nx activate-key`

If the user passes the key, it does what it does now.

If the user does not pass the key, show them the registration flow to get a key